animata/card/card-spread.tsx

```
@@ -29,60 +29,56 @@ function RemodelNotes() {
               e.preventDefault();
             }}
-            className={cn(
-              "transition duration-500 ease-in-out",
-              {
-                absolute: !isExpanded,
-                "origin-bottom": !isExpanded,
-              },
-              !isExpanded && item.rotationClass,
-              isExpanded && item.revealClass,
-            )}
+            tabIndex={0}
```

Biome (2.4.15)

[error] The HTML element div is non-interactive. Do not use tabIndex.

(lint/a11y/noNoninteractiveTabindex)